benchmark/kernels/fused_moe_triton/tuning_fused_moe_triton.py

```diff
@@ -18,6 +18,7 @@
     get_default_config,
     get_moe_configs,
 )
+from sglang.srt.utils import is_hip
```

Comment:
Please ensure it caches is_hip instead of calling every time.

Reply:
to something like:

`_is_hip = is_hip()`

then use `_is_hip` globle in following file